util/build_configs/cray-internal/generate-modulefile.bash

```diff
@@ -68,17 +68,14 @@ proc ModulesHelp { } {
 }
 
 conflicts cray-mpich2 < 7.0.0
-conflicts xt-mpich2 < 6.0.0
 conflict chapel
 
 ##
 ## Now, we're going to attempt to get to the right build type
 ###
 #
 set network aries
-if { [ info exists env(XTPE_NETWORK_TARGET) ] } {
```

Comment:
Just FYI I left this in in https://github.com/chapel-lang/chapel/pull/16247 since `XTPE_NETWORK_TARGET` is still set on modern machines, but I'm on board with removing it